LSP-metals.sublime-settings

```diff
@@ -13,6 +20,30 @@
     "-Xms100m"
   ],
 
-  // Optional path to the Java home directory. Requires reloading sublime.
-  "java_home": ""
+  // The startup command (ST4 only)
+  "command": [
```

Comment:
this shouldn't leak to the user setting. One would care only about `server_properties`